oml/lightning/callbacks/metric.py

```diff
@@ -101,11 +102,17 @@ def _log_images(self, pl_module: pl.LightningModule) -> None:
             log_str = f"{LOG_IMAGE_FOLDER}/{metric_log_str}"
             if isinstance(pl_module.logger, NeptuneLogger):
                 pl_module.logger.experiment[log_str].log(File.as_image(fig))
+            elif isinstance(pl_module.logger, WandbLogger):
+                fig.canvas.draw()
+                fig_img = np.array(fig.canvas.renderer.buffer_rgba())[..., :3]
+                pl_module.logger.log_image(images=[fig_img], key=metric_log_str)
+                plt.close(fig)
             elif isinstance(pl_module.logger, TensorBoardLogger):
                 fig.canvas.draw()
                 data = np.frombuffer(fig.canvas.tostring_rgb(), dtype=np.uint8)
                 data = data.reshape(fig.canvas.get_width_height()[::-1] + (3,))
                 pl_module.logger.experiment.add_image(log_str, np.transpose(data, (2, 0, 1)), pl_module.current_epoch)
+                plt.close(fig)
```

Comment:
It looks like code duplication. The logic of getting image from figure could be extracted into a function, at used here.

Reply:
@nik-fedorov could you take care of this with other todos in the next PR, please?

Reply:
Ok :)